ECMAScript/react-native/src/components/QuilttConnector.tsx

```diff
@@ -33,7 +34,7 @@ export const QuilttConnector = ({
   const webViewRef = useRef<WebView>(null)
   const { session } = useQuilttSession()
   oauthRedirectUrl = encodeURIComponent(oauthRedirectUrl)
-  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native` // @todo append version from package.json
+  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native&version=${version}`
```

Comment:
To better differentiate from the API version, can we rename this `sdk_version`?

Reply:
```suggestion
  const connectorUrl = `https://${connectorId}.quiltt.app/?mode=webview&oauth_redirect_url=${oauthRedirectUrl}&sdk=react-native& sdk_version =${version}`
```

Reply:
yeah i think this works

Reply:
Switched to `agent=react-native-${version}` based on @sirwolfgang's input.
https://linear.app/quiltt/issue/Q-679/track-version-in-react-and-core-sdk#comment-add19507